ruby/gilded_rose.rb

```diff
@@ -0,0 +1,68 @@
+class GildedRose
+
+  def initialize(items)
+    @items = items
+  end
+
+  def update_quality()

```

Comment:
Assignment Branch Condition size for update_quality is too high. [62.36/15]<br>Omit the parentheses in defs when the method doesn't accept any arguments.